### ISSUE TITLE
captilised variables error on puppet not with lint

### ISSUE THREAD

**Issue by AutomatedTester** [rodjek/puppet-lint#158]
if I do 

```
class {
    "my-class":
         ZOMG_PUPPET => 'baz';
}
```

I get `Could not parse for environment production: Syntax error at 'ZOMG_PUPPET'; expected '}'` from puppet but lint does not show the error
